woocommerce/woocommerce
# Add onboarding support for new countries in WCPay

### All Submissions:






### Changes proposed in this Pull Request:



Closes #36904.

This PR adds onboarding support for 12 new countries in WCPay:

Before this PR:
```
[ 'US', 'PR', 'AU', 'CA', 'DE', 'ES', 'FR', 'GB', 'IE', 'IT', 'NZ', 'AT', 'BE', 'NL', 'PL', 'PT', 'CH', 'HK', 'SG' ].length
19 
```

After this PR:
```
[ 'US', 'PR', 'AU', 'CA', 'CY', 'DE', 'DK', 'EE', 'ES', 'FI', 'FR', 'GB', 'GR', 'IE', 'IT', 'LU', 'LT', 'LV', 'NO', 'NZ', 'MT', 'AT', 'BE', 'NL', 'PL', 'PT', 'CH', 'HK', 'SI', 'SK', 'SG' ].length
31 
```

**This PR targets the next version of WC `7.6.0`**

WCPay PR for info: https://github.com/Automattic/woocommerce-payments/pull/5536




### Testing instructions:

To trigger the OBW fallback, disable marketplace suggestions:  
Settings > [WooCommere.com](http://woocommere.com/) > untick Display suggestions within WooCommerce

1. Get this PR locally.
2. Change your store location to one of the countries in this list (eg. Denmark, Finland, etc.).
3. Trigger the WooCommerce onboarding wizard.
4. Make sure WC Payments is showing as the suggested payment gateway in the "Set up payments" step.

### Other information:




### FOR PR REVIEWER ONLY:
